src/python/tritonfrontend/_c/__init__.pyi

```diff
@@ -0,0 +1 @@
+# TODO: NEED TO ADD FUNCTION STUBS
```

Comment:
What's needed here? CC @nnshah1

Reply:
For the in process c-bindings, I used mypy to generate the stub information, after installation.

never fully automated, but the general procedure I found useful:

1) install wheel with bindings
2) if you do any aliasing in __init__.py (types, etc,) - remove this to avoid them interfering with stub generation
2) run stubgen -p .... <module name>


# Note: this file was generated using mypy with an empty __init__.py
# file in the tritonserver package directory to avoid any renaming /
# aliasing done by the wrapper
#
# mypy 1.8.0 (compiled: yes)
#
# stubgen -p tritonserver._c
#
# Todo: add stub generation to build process

Reply:
Thanks for detailed comments. Just finished generating the stubs!

Reply:
for any generated files let's add the same note for a breadcrumb for future automation - or if you have the automated part of build - we can back port to python in process python api build as well.